last_seen.py

```diff
@@ -0,0 +1,27 @@
+from brutal.core.plugin import BotPlugin, cmd, event
+from time import strftime, gmtime
+
+class LastSeen(BotPlugin):
+    def setup(self, *args, **kwargs):
+        self.seen = self.open_storage('last_seen')
+
+    @event
+    def seen_evt(self, event):
+        events = ['quit', 'part', 'kick', 'join', 'rename', 'topic']
+        if event.event_type in events:
+            nick = event.meta['nick']
+            time = strftime("%H:%M %d.%m.%Y", gmtime())
+            self.seen[nick] = time
```

Comment:
@JakubNvk regarding @Adman's note -- that will give you the ability to decide what to do with a raw date afterwards. In the current setup you can only show the string you have saved.